### PR TITLE
ADD: macOS catalina support for scanning in is it my address

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tdbcxrwjtcp7z7486xpmdy1ej0

[download apk](http://filestorage.bluewallet.io:1488/1608326962.apk)

**Review by ncoelho — APPROVED** [review]

File: blue_modules/environment.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { getSystemName } from 'react-native-device-info';
+
+const isMacCatalina = getSystemName() === 'Mac OS X';
+
+module.exports.isMacCatalina = isMacCatalina;
```
Comment (Overtorment) [inline blue_modules/environment.js:5]: gj for code reuse

File: blue_modules/environment.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { getSystemName } from 'react-native-device-info';
+
+const isMacCatalina = getSystemName() === 'Mac OS X';
```
Comment (Overtorment) [inline blue_modules/environment.js:3]: i hope its not an async call

**Review by Overtorment — APPROVED** [review]

File: blue_modules/environment.js (hunk 2 above)
Comment (marcosrdz) [reply blue_modules/environment.js:3]: it returns a string, not a promise.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/366d8j703dt6rnbuvxpt4peh48

[download apk](http://filestorage.bluewallet.io:1488/1608421320.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mpe40twqudn4p3rduey6fg4ep8

[download apk](http://filestorage.bluewallet.io:1488/1608657358.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/nfad708narfcd6ycd1x6he00pw

[download apk](http://filestorage.bluewallet.io:1488/1608838977.apk)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**